Code/Source/RobotImporter/URDF/URDFPrefabMaker.cpp

```diff
@@ -133,13 +139,17 @@ namespace ROS2
 
     void URDFPrefabMaker::AddRobotControl(AZ::EntityId rootEntityId)
     {
-        // TODO - check for RigidBody
```

Comment:
@adamdbrw - I am not sure if I resolved this TODO correctly.
Since this way creates components in a safe manner, there is no need for such a check (I added the required rigid body service to ROS2RobotControlComponent)